src/Css.ts

```diff
@@ -3966,12 +3966,15 @@ class CssBuilder<T extends Properties> {
   }
 
   // visuallyHidden
-  /** Sets `position: "absolute"; overflow: "hidden"; clip: "inset(50%)"; clipPath: ""; border: 0; height: "1px"; margin: "-1px"; width: "1px"; padding: 0; whiteSpace: "nowrap"`. */
+  /** Sets `position: "absolute"; overflow: "hidden"; clip: "inset(50%)"; clipPath: ""; border: 0; height: "1px"; margin: "-1px"; width: "1px"; padding: 0; whiteSpace: "nowrap"; opacity: 0`. */
```

Comment:
curious: This comment doesn't seem add additional value over reading the code 🤔

Reply:
These JSDoc comments are automatically generated by Truss. They make it easier to hover over a Truss property in your IDE and see what it's setting.